fvm/src/kernel/default.rs

```diff
@@ -672,24 +672,23 @@ where
         &self.call_manager.context().network_context.base_fee
     }
 
-    fn tipset_timestamp(&self) -> u64 {
+    fn tipset_timestamp(&self) -> i64 {
         self.call_manager.context().network_context.timestamp
     }
 
-    fn tipset_cid(&self, epoch: i64) -> Result<Option<Cid>> {
+    fn tipset_cid(&self, epoch: ChainEpoch) -> Result<Cid> {
         if epoch < 0 {
```

Comment:
So... We _could_ interpret this as "relative to the current epoch". Thoughts? Disgust?

Reply:
uhm, lets just make it absolute and not bother with edge cases.

I am fine with returning Result here instead of an Option, esp. if this happens through an extern in the near future.